python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -5,7 +5,7 @@
     import unittest
```

Comment:
I was under the impression this simply wasn't called if you were running python 2.6 (if it works with unittest2 then I'm happy to add this as it fixes some problems)